doc/developers/advanced_installation.rst

```diff
@@ -142,24 +142,39 @@ Mac OSX
 -------
 
 The default C compiler, Apple-clang, on Mac OSX does not directly support
-OpenMP. The first solution to build scikit-learn is to install another C
-compiler such as gcc or llvm-clang. Another solution is to enable OpenMP
-support on the default Apple-clang. In the following we present how to
-configure this second option.
+OpenMP. We present two solutions to enable OpenMP support (you need to do only
+one).
 
-You first need to install the OpenMP library::
+- Install another C compiler.
 
-    brew install libomp
+    If you use the conda package manager, you can install C compilers from the
+    conda-forge channel::
 
-Then you need to set the following environment variables::
+        conda install conda-forge::compilers
 
-    export CC=/usr/bin/clang
-    export CXX=/usr/bin/clang++
-    export CPPFLAGS="$CPPFLAGS -Xpreprocessor -fopenmp"
-    export CFLAGS="$CFLAGS -I/usr/local/opt/libomp/include"
-    export CXXFLAGS="$CXXFLAGS -I/usr/local/opt/libomp/include"
-    export LDFLAGS="$LDFLAGS -L/usr/local/opt/libomp/lib -lomp"
-    export DYLD_LIBRARY_PATH=/usr/local/opt/libomp/lib
+    You then need to set the following environment variables::
+
+        export CFLAGS="$CFLAGS -I<path>/include"
+        export LDFLAGS="$LDFLAGS -L<path>/lib -Wl,rpath,<path>/lib"
+
+    assuming <path> is the path to your conda environment (it should be
```

Comment:
assuming -> where

Reply:
it should be -> usually

Reply:
Better use the `$CONDA_PREFIX` env var as suggested above.

Reply:
Maybe add a note stating that this env var needs to be configured again for any new bash session.

It depends on the value `$CONDA_PREFIX` and therefore needs to be adjusted for users who use several conda environment with the `conda activate envname` command.

Reply:
Ah nice ! I thought there was an env var for that but did not recall its name :)

Reply:
`$CONDA_PREFIX` is defined as long as one of the environments are "activate". In other words, they use `conda init` so that the `base` environment is active when their shell starts. I believe this is the recommend way of install conda.

On a side note, users that just add `$HOME/anaconda3/bin` to their `PATH` will not have `$CONDA_PREFIX` defined.

Reply:
but $CONDA_PREFIX is automatically set at each new bash session. If you haven't activated any env, it will point to base. But I assume that when you do `conda install compilers` you're installing it on the desired env.

Reply:
Maybe I can just add a note like "If $CONDA_PREFIX is not set, replace it by the path to the desired conda environment" ?

Reply:
`$CONDA_PREFIX` is set as long as one of the conda envs are activated from `.bashrc` or `.bash_profile`.

In my setup, I do not have `$CONDA_PREFIX` defined on startup. I do a very untraditional thing of setting:

```
export PATH=$HOME/anaconda3/bin:$PATH
```

I think most people that installs anaconda will go with the default which adds 

```
. "/Users/thomasfan/anaconda3/etc/profile.d/conda.sh"
```

to their shell rc file, which by default activate `base` on startup.

In the end, I would prefer to use `$CONDA_PREFIX` and assume users did a default installation of anaconda.

Reply:
I updated this part. Let me know what you think

Reply:
I think it is good. During sprints, people with default installations of anaconda should not run into issues with `CONDA_PREFIX`.